src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html

```diff
@@ -51,6 +51,22 @@
     </ul>
 </app-page-options-section>
 
+<app-page-options-section *ngIf="associatedPullRequests.length > 0" sectionName="Associated Pull Requests">
```

Comment:
Will there be more than one pull request for a revision? A PR based revision is always associated with only one pull request. right?

Reply:
Yes that is very much the case

Reply:
Can you send me an example where a revision contains more than one PR? That should not be the case. A pull request revision should associate with only one PR.

Reply:
Actually, I was agreeing with your previous assertion. the PR revision is created via one PR. This is just a check to make sure that the option is not shown when there is no associatedPullRequests